config/initializers/assets.rb

```diff
@@ -11,4 +11,36 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
-# Rails.application.config.assets.precompile += %w( admin.js admin.css )
+Rails.application.config.assets.precompile += %w(
```

Comment:
I'm not sure, ini bakal masukin semuanya ke asset pipeline yah?

Reply:
Yup